bubblesub/tests/test_run.py

```diff
@@ -14,12 +14,18 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+import os
+import signal
 import subprocess
 import typing as T
 
+SERVER_NUM = 99
+
 
 def test_run() -> None:
     try:
-        subprocess.run("bubblesub", timeout=4, check=True)
+        subprocess.run(["xvfb-run", "bubblesub"], timeout=4, check=True)
     except subprocess.TimeoutExpired:
+        pid = int(open("/tmp/.X%s-lock" % SERVER_NUM).read().strip())
```

Comment:
Please use `tempfile.gettempdir`, `Path` and `f''` formatting.
Something like 

```python
tmp_dir = Path(tempfile.gettempdir())
lock_file = tmp_dir / f'.X{SERVER_NUM}-lock'
pid = int(lock_file.read_text().strip())
```

Reply:
Done, thanks!